journal/src/test/scala/com/evolutiongaming/kafka/journal/JournalSpec.scala

```diff
@@ -31,10 +31,10 @@ class JournalSpec extends AnyWordSpec with Matchers {
   import JournalSpec._
 
   // TODO add test when Kafka missing it's tail comparing to eventual
-  def testF[F[_] : Monad](withJournal: (SeqNrJournal[F] => F[Assertion]) => Assertion): Unit = {
```

Comment:
could you please try to revert formatting changes here? it would simplify review.